e2e/cypress/integration/plugins/systemconsole/plugin_upload_spec.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I forgot to mention to please move this file into `/cypress/integration/system_console/plugin_upload_spec.js`